apps/journeys-admin-e2e/src/pages/card-level-actions.ts

```
@@ -968,16 +967,39 @@ export class CardLevelActionPage {
+
+    let found = false
+    for (const selector of chatIconSelectors) {
+      try {
+        await expect(
+          this.page.frameLocator(this.journeyCardFrame).locator(selector)
+        ).toBeVisible({ timeout: 3000 })
+        found = true
+        break
+      } catch (error) {
+        // Try next selector
+        continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)